imports/plugins/core/checkout/client/components/completedOrderItem.js

```diff
@@ -18,7 +18,7 @@ const CompletedOrderItem = ({ item, handleDisplayMedia }) => {
   }
   return (
     <div className="row order-details-line">
-      <div className="order-details-media"><img src={image} /></div>
+      <div className="order-details-media"><img src={image} alt="" /></div>
```

Comment:
Should we put something int he alt tag, instead of leaving it blank?

Reply:
No, as the image is for decoration. https://www.w3.org/WAI/tutorials/images/decorative/ namely these two bullet points:

- Illustrative of adjacent text but not contributing information (“eye-candy”);
- Identified and described by surrounding text.